test/controllers/static_pages_controller_test.rb

```diff
@@ -0,0 +1,30 @@
+require 'test_helper'
```

Comment:
Please remove this auto generated test files, you don't need to care about testing right now.